x-pack/packages/kbn-elastic-assistant/impl/assistant/settings/assistant_settings_management.tsx

```diff
@@ -121,10 +113,20 @@ export const AssistantSettingsManagement: React.FC<Props> = React.memo(
 
     useEffect(() => {
       if (selectedConversation != null) {
-        setSelectedConversation(conversationSettings[selectedConversation.title]);
+        setSelectedConversation(
+          // conversationSettings has title as key, sometime has id as key
+          conversationSettings[selectedConversation.id] ||
+            conversationSettings[selectedConversation.title]
```

Comment:
@YulNaumenko Is this the do with the existing issue?
conversationSettings has title as key, sometime has id as key